apps/ledger-live-mobile/src/screens/ImportAccounts/DisplayResultItem.tsx

```diff
@@ -56,3 +19,38 @@ const styles = StyleSheet.create({
     marginLeft: 8,
   },
 });
+
+type Props = {
+  account: Account;
+  mode: ImportItem["mode"];
+  checked: boolean;
+  importing: boolean;
+  onSwitch: (checked: boolean, account: Account) => void;
+};
+
+const DisplayResultItem = ({ account, mode, checked, importing, onSwitch }: Props) => {
+  const _onSwitch = () => {
```

Comment:
Can you rename it without _, please? As _ prefix is a convention for private functions/ attributes.
You can rename it as `onSwitchAccount` or `handleSwitch` etc...